docs/tutorials/customize_runtime.md

```diff
@@ -0,0 +1,321 @@
+# Tutorial 5: Customize Runtime Settings
```

Comment:
could you explain the `Runtime` a little bit?

Reply:
A description is given as the following.